src/components/Collapse/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import { ReactComponent as LargeArrowIcon } from 'assets/revamp-icon/arrow-large.svg';
+import clsx from 'clsx';
+
+export type itemType = {
+  key: string;
+  label: string;
+  children: React.ReactNode;
+};
+
+type IProps = {
+  items: itemType[];
+  defaultActiveKey?: string[];
+};
+
+const CollapseItem = ({ item, isOpened }: { item: itemType; isOpened?: boolean }) => {
```

Comment:
move this into another file, since the folder structure now is Collapse/index.tsx. you can create Collapse/CollapseItem/index.tsx for your collapse item

Reply:
Moved